### DISCUSSION TITLE
Product Bundles

### DISCUSSION THREAD

**Discussion by glennjacobs** [lunarphp/lunar#1029] [Feature Requests]

**Comment by thejawker** [comment]
Thanks for the project, really digging it. Would definitely benefit from a native Product Bundle in lunar.

**Comment by Navi2016** [comment]
If i understand correctly, Lunar does allow to create Products and Variants now, but you can't create a bundle to sell?

For example a product Cable with variant 2ft and 3ft. And a product Phone.

You can't sell a Phone + 2ft cable bundle?

**Reply by glennjacobs** [reply]
So Lunar supports products and variants, but we don't have anything for bundles as of yet. Bundles are normally the responsibility of an inventory system, hence not having it. But it is certainly something we want to address once out of beta.

**Reply by Navi2016** [reply]
I have it in my current (very old custom) shop and looking to move to lunar.

It's not super complicated code wise, the bundle is simply a variant with multiple variants in it and has it's own price, but no own stock or weight. (they are calculated from the variants in the bundle)

Display the bundle as a variant for a product, and when sold, subtract stock from the variants in the bundle.

I would be willing to sponsor some $ for this functionality in Lunar if needed.
